packages/blockchain-wallet-v4-frontend/src/modals/Bch/RequestBch/index.js

```diff
@@ -83,7 +83,8 @@ class RequestBchContainer extends React.PureComponent {
           handleSubmit={this.handleSubmit}
           type={value.type}
           closeAll={closeAll}
-          excludeLockbox={value.excludeLockbox}
+          excludeLockbox={value.excludeLocokbox}
+          receiveAddressFromProps={this.props.receiveAddressFromProps}
```

Comment:
Can we not do ownProps.receiveAddress || value.receiveAddress. https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/1313/files#diff-d204961bca7733db5113a359bd5ea5ebR81

That way you avoid touching selectors

Reply:
Nice catch, will implement this